Learning-Tools-Interoperability.md

```diff
@@ -0,0 +1,67 @@
+# Learning Tools Interoperability (LTI)
+
+MarkUs integrates with other Learning Management Systems via the LTI standard.
+Currently MarkUs supports the following LMS platforms:
+
+- Canvas
+
+## For LMS Administrators
+
+### Installing MarkUs on an LMS
+
+Each LMS implements their own LTI integration process.
+Typically, only administrators can add LTI integrations.
+
+### Canvas
+
+To add MarkUs to a Canvas instance, see their page on
+[configuring LTI keys](https://community.canvaslms.com/t5/Admin-Guide/How-do-I-configure-an-LTI-key-for-an-account/ta-p/140).
+
+MarkUs can be added via a configuration URL. The configuration URL is available at  /lti_deployments/get_canvas_config
+
+## For Instructors
+
+Once installed in your course, a 'Launch Markus' page will appear in your
+course's navigation (disabled by default), and needs to be added to the navigation:
+
+![Canvas MarkUs Navigation](images/canvas-markus-nav.png)
+
+If you believe MarkUs should be installed in your course but it does not appear,
+contact your LMS admistrators.
+
+> **NOTE:**
+> The additional navigation item will only be visible to instructors and
+> administrators (not students)
+
+### Associating your LMS Course with your MarkUs course
+
+Once MarkUs is configured with your LMS, an association between your
```

Comment:
You use LMS and Canvas interchangeably. Please make sure that we're only referring to Canvas in these sections not LMS in general